packages/widget/src/types/events.ts

```diff
@@ -30,6 +31,7 @@ export type WidgetEvents = {
   reviewTransactionPageEntered?: Route;
   walletConnected: WalletConnected;
   widgetExpanded: boolean;
+  page: string;
```

Comment:
Let's create a union type from `navigationRoutes` constant instead of string that lists all relevant pages we would like to track, except maybe `home`? Let me please know what do you think :)

Reply:
Good idea! I have re-used the existing `NavigationRouteType`.

Just not sure with this: To silence ts, I have used it that way here:
![image](https://github.com/user-attachments/assets/81a2b77a-2d56-4991-b0b2-a8b5420bdfe4)